### PR TITLE
Publicize the Modifier classes

### PR DESCRIPTION
**Pull request by EltonZhang777**

### CODE REVIEW

File: Blasphemous.Framework.Levels/Modifiers/TrapModifiers.cs [hunk 1]
```diff
@@ -2,8 +2,14 @@
 
 namespace Blasphemous.Framework.Levels.Modifiers;
 
-internal class SpikeModifier : IModifier
+/// <summary>
+/// modifier for spike objects
+/// </summary>
+public class SpikeModifier : IModifier
 {
+    private Vector2 _colliderSize;
```
Comment (BrandenEK) [inline Blasphemous.Framework.Levels/Modifiers/TrapModifiers.cs]: Make this field readonly

File: Blasphemous.Framework.Levels/LevelFramework.cs [hunk 2]
```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
...
+        // spikes
+        provider.RegisterObjectCreator("spikes-wasteland", new ObjectCreator(
             new SceneLoader("D01Z03S01_DECO", "MIDDLEGROUND/AfterPlayer/Spikes/{0}"),
             new SpikeModifier()));
 
         // Misc.
         provider.RegisterObjectCreator("lantern", new ObjectCreator(
             new SceneLoader("D20Z01S02_LOGIC", "LOGIC/INTERACTABLES/Chain Hook"),
             new NoModifier("Lantern")));
+        provider.RegisterObjectCreator("bell-face", new ObjectCreator(
```
Comment (BrandenEK) [inline Blasphemous.Framework.Levels/LevelFramework.cs]: Have you verified that these bells work as expected?

File: Blasphemous.Framework.Levels/LevelFramework.cs [hunk 3]
```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
...
+            new SceneLoader("D03Z02S06_LOGIC", "TRAPS/TRAP_SHOCK_ENEMY"),
+            new NoModifier("Face bell")));
+        provider.RegisterObjectCreator("bell-iron", new ObjectCreator(
+            new SceneLoader("D03Z02S06_LOGIC", "TRAPS/TRAP_SHOCK_CHAIN_REACTION"),
+            new NoModifier("Iron bell")));
+        provider.RegisterObjectCreator("ladder-jondo", new ObjectCreator(
```
Comment (BrandenEK) [inline Blasphemous.Framework.Levels/LevelFramework.cs]: You added a ladder object but there is no class called "LadderModifier".  Go ahead and remove this, as I dont think ladders belong in the framework yet

File: Blasphemous.Framework.Levels/LevelFramework.cs [hunk 4]
```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
             new SceneLoader("D17Z01S10_LOGIC", "LOGIC/INTERACTABLES/{0}"),
             new BloodModifier()));
 
-        // Traps
-        provider.RegisterObjectCreator("spikes", new ObjectCreator(
+        // spikes
```
Comment (BrandenEK) [inline Blasphemous.Framework.Levels/LevelFramework.cs]: Comments should start with an uppercase letter.  This also applies to the comments in the documentation for each modifier, capitalize those as well

**Review by BrandenEK — CHANGES_REQUESTED** [review]

File: Blasphemous.Framework.Levels/LevelFramework.cs (hunk 2 above)
Comment (EltonZhang777) [reply Blasphemous.Framework.Levels/LevelFramework.cs]: I'm still experimenting on whether the face bells works as intended, but I'm sure that the iron bells works as intended. I'll delete the `bell-face` creator for now.

**Review by EltonZhang777 — COMMENTED** [review]
Thanks, all issues are addressed in the commit `misc. cleanup`.

**Review by BrandenEK — APPROVED** [review]
Yep thanks for making those changes.  Just trying to keep code and documentation consistent

**Pull request merged**